### PR TITLE
Bump `open-autonomy` and `open-aea`

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Bumps `open-autonomy` and `open-aea`.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [x] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: packages/valory/contracts/offchain_aggregator/contract.yaml [hunk 1]
```diff
@@ -18,5 +18,9 @@ class_name: OffchainAggregatorContract
 contract_interface_paths:
   ethereum: build/VerySimpleOffchainAggregator.json
 dependencies:
+  eth-abi:
+    version: ==2.2.0
```
Comment (Adamantios) [inline packages/valory/contracts/offchain_aggregator/contract.yaml:22]: Why is this required by `autonomy check-packages` after `0.3.4`? This is not used anywhere in the repo. Only from `web3`.

**Review by 0xArdi — APPROVED** [review]

File: packages/valory/contracts/offchain_aggregator/contract.yaml (hunk 1 above)
Comment (DavidMinarsch) [reply packages/valory/contracts/offchain_aggregator/contract.yaml:22]: Might be an error in the logic. Suggest you open an issue. Only direct imports should be tagged

**Review by DavidMinarsch — APPROVED** [review]

File: packages/valory/contracts/offchain_aggregator/contract.yaml (hunk 1 above)
Comment (Adamantios) [reply packages/valory/contracts/offchain_aggregator/contract.yaml:22]: Opened https://github.com/valory-xyz/open-aea/issues/427.
